var/spack/repos/builtin/packages/muparser/package.py

```diff
@@ -6,7 +6,7 @@
 from spack.package import *
 
 
-class Muparser(Package):
+class Muparser(CMakePackage, Package):
```

Comment:
When a package inherits multiple build systems, it should define a `build_system` variant; otherwise the phase inheritance/application is uncertain. `quantum-espresso` is a package that can optionally build with `CMake` or with a custom script, so I think it would be worth using that as an example for how to define the variant (it also shows restricting the available versions that use `cmake`).